Rscript/alignTargetedRuns_cli.R

```diff
@@ -0,0 +1,220 @@
+#!/bin/usr/Rscript
+
+#' Command line interface to alignTargetedRuns
+#'
+#' This function is to be used via the command line, to run alignTargetedRuns
+#' @author Justin Sing, \email{justinc.sing@mail.utoronto.ca}
+#'
+#' ORCID: 0000-0003-0386-0092
+#'
+#' License: (c) Author (2019) + GPL-3
+#' Date: 2021-08-27
+#' @inheritParams alignTargetedRuns
+#' @seealso \code{\link{alignTargetedRuns}
+#' @examples
+#' Rscript alignTargetedRuns_cli.R --dataPath=/data/osw/ --params=context:experiment-wide,maxFdrQuery:0.01
+#' @references Gupta S, Ahadi S, Zhou W, Röst H. "DIAlignR Provides Precise Retention Time Alignment Across Distant Runs in DIA and Targeted Proteomics." Mol Cell Proteomics. 2019 Apr;18(4):806-817. doi: https://doi.org/10.1074/mcp.TIR118.001132 Epub 2019 Jan 31.
+alignTargetedRuns_cli <- function(){
+  # Get Arguments passed to command line
+  args <- commandArgs(trailingOnly = TRUE)
+  # Extract Passed Command Line Arguments
+  args <- parseArgs(args)
+  # Register BioCParallel workers if being used
+  if (("regBioCP" %in% names(args))){
+    message("INFO: Registering BiocParallel...")
+    eval(parse(text = args[["regBioCP"]]))
+  }
+  # Main call to DIAlignR's alingment of Targeted Runs
+  DIAlignR::alignTargetedRuns(dataPath=args[["dataPath"]], outFile=args[["outFile"]], params=args[["params"]], oswMerged=args[["oswMerged"]],
```

Comment:
Shouldn't call DIAlignR from the same package.

Reply:
We would have to call `library("DIAlignR")` then, because the Rscript is external from the main package itself. Is that fine?